<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vl_global_vl">
     <item>
      <layout class="QHBoxLayout" name="hl_title">
       <item>
        <widget class="QLabel" name="lbl_title">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>22</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>GPIO OSC Server</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vl_set_status">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lbl_set_status">
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>GPIO Set Status</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pb_led_17">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>GPIO 17 - LED</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_blinders_27">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>GPIO 27 - BLINDERS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_FM_1_22">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>GPIO 22 - FIRE MACHINE 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_FM_2_23">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>GPIO 23 - FIRE MACHINE 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_FM_3_24">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>GPIO 24 - FIRE MACHINE 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_enable_all">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>ENABLE ALL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_disable_all">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>DISABLE ALL</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="lbl_get_status">
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>GPIO Get Status</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl_GPIO17_LED">
         <item>
          <widget class="QLabel" name="lbl_GPIO17_LED">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>GPIO 17 status LED:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_GPIO17_LED_status">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>OFF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl_GPIO27_blinders">
         <item>
          <widget class="QLabel" name="lbl_GPIO27_blinders">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>GPIO 27 status Blinders:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_GPIO27_blinders_status">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>OFF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl_GPIO22_FM_1">
         <item>
          <widget class="QLabel" name="lbl_GPIO22_FM_1">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>GPIO 22 status Fire Machine 1:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_GPIO22_FM_1_status">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>OFF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl_GPIO23_FM_2">
         <item>
          <widget class="QLabel" name="lbl_GPIO23_FM_2">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>GPIO 23 status Fire Machine 2:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_GPIO23_FM_2_status">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>OFF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl_GPIO24_FM_3">
         <item>
          <widget class="QLabel" name="lbl_GPIO24_FM_3">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>GPIO 24 status Fire Machine 3:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_GPIO24_FM_3_status">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>OFF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
